### PR TITLE
add corpsman gloves to the vendor

### LINKED ISSUE #1793: Combat Sterile Gloves

### PR DESCRIPTION
**Pull request by Dutch-VanDerLinde**
resolves #1793

### CODE REVIEW

File: Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/bundles.yml [hunk 1]
```diff
@@ -41,7 +41,6 @@
     - CMArmorHelmetM10Medic # TODO RMC14 M4
     - CMVendorHeadsetSquadCorpsman
     - CMJumpsuitMarineMedic
-    - CMHandsBlackMarine
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/bundles.yml:44]: bundle description is wrong after this change

**Review by QuietlyWhisper — CHANGES_REQUESTED** [review]

**Comment by Dutch-VanDerLinde** [comment]
fixed

File: Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Squad/medic.yml [hunk 2]
```diff
@@ -211,7 +211,8 @@
       - id: CMVendorBundleSquadMedicalApparel
       - id: CMMRE
```
Comment (DrSmugleaf) [inline Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Squad/medic.yml:212]: Small nitpick but in 13 the gloves are above the MRE
```suggestion
      - id: CMVendorBundleSquadMedicalApparel
      - id: RMCHandsLatexMarine
        name: combat sterile gloves
      - id: CMMRE
```

File: Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Squad/medic.yml [hunk 3]
```diff
@@ -211,7 +211,8 @@
...
       #- id: CMMap # TODO: Make a map
-    #- id: CMCombatSterileGloves
+      - id: RMCHandsLatexMarine
+        name: combat sterile gloves
```
Comment (DrSmugleaf) [inline Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Squad/medic.yml:215]:
```suggestion
```

**Review by DrSmugleaf — APPROVED** [review]

**Pull request merged**